docs/content/docs/tutorials/ec2-example.md

```diff
@@ -0,0 +1,451 @@
+  Owner ID:                <ID>
+  State:                   available
+  Vpc ID:                  vpc-<ID>
+Events:                    <none>
+```
+ 
+* The **VPC** resource synced successfully and is available. Note the `vpc-<ID>`.
+ 
+* Create the **Subnet** using `vpc-<ID>`, the provided YAML, and `kubectl apply`:
+```
+cat <<EOF > subnet.yaml
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: Subnet
+metadata:
+  name: subnet-ack-test
+spec:
+  cidrBlock: 10.0.0.0/20
+  vpcID: vpc-<ID>
+EOF
+ 
+kubectl apply -f subnet.yaml
+```
+ 
+* Check the **Subnet** availability and ID using `kubectl describe`:
+```
+> kubectl describe subnets
+...
+Status:
+  Ack Resource Metadata:
+    Arn:                       arn:aws:ec2:us-west-2:<ID>:subnet/subnet-<ID>
+    Owner Account ID:          <ID>
+    Region:                    us-west-2
+  Available IP Address Count:  4091
+  Conditions:
+    Last Transition Time:           2022-10-12T17:36:53Z
+    Message:                        Resource synced successfully
+    Reason:
+    Status:                         True
+    Type:                           ACK.ResourceSynced
+  Default For AZ:                   false
+  Map Customer Owned IP On Launch:  false
+  Owner ID:                         <ID>
+  Private DNS Name Options On Launch:
+  State:      available
+  Subnet ID:  subnet-0f0eb8efb37eb43b0
+Events:       <none>
+```
+ 
+* Delete the resources:
+  * `kubectl delete -f subnet.yaml`
+  * `kubectl delete -f vpc.yaml`
+ 
+ 
+Both resources were successfully deployed, managed, then deleted by their respective controllers. Although contrived, this example highlights how easy it can be to deploy AWS resources via YAML files and how it feels like managing any other K8s resource. 
+ 
+In this example, waiting for the `vpc-ID` in order to create the **Subnet** was sub-optimal; the workflow didn't feel declarative. The next example alleviates these concerns using a single manifest (YAML) to deploy the entire network topology.
+ 
+ 
+### Create a VPC Workflow
+ 
+This section details the steps to create a network topology consisting of multiple, connected resources from a single manifest file. The following resources will be present in said manifest:
+* 1 VPC
+* 1 Instance
+* 1 Internet Gateway
+* 1 NAT Gateways
+* 1 Elastic IPs
+* 2 Route Tables
+* 2 Subnets (1 Public; 1 Private)
+* 1 Security Group
+ 
+ 
+The VPC is connected to the internet through an Internet Gateway. The NAT Gateway is created in the public Subnet with an associated Elastic IP. The Instance is deployed into the private Subnet which can connect to the internet using the NAT Gateway in the public Subnet. Lastly, one Route Table (public) will contain a route to the Internet Gateway while the other Route Table (private) contains a route to the NAT Gateway.
+
+* Deploy the resources using the provided YAML and `kubectl apply -f vpc-workflow.yaml`:
+ 
+```
+apiVersion: ec2.services.k8s.aws/v1alpha1
+kind: VPC
+metadata:
+  name: ack-vpc
```

Comment:
nit: All of these resource names start with `ack-` which is redundant. Perhaps rename to `tutorial-` instead?